central/deployment/datastore/datastore_bench_test.go

```diff
@@ -26,6 +27,8 @@ import (
 )
 
 func BenchmarkSearchAllDeployments(b *testing.B) {
+	pgtest.SkipIfPostgresEnabled(b)
+
 	ctx := sac.WithGlobalAccessScopeChecker(context.Background(),
```

Comment:
Either we need to migrate this or just delete it

Reply:
There is TODO JIRA for it

Reply:
Can you put a todo comment in the code then like you did for the others?